src/lib.rs

```diff
@@ -398,7 +451,12 @@ pub fn list_cat(options: Options<CatFilter>) -> Result<Paged<Cat>, Error> {
         .map_or_else(CatFiltersInput::default, |filter| filter.into());
     let pagination = options.pagination.unwrap_or_default();
     let sort: Option<Vec<Option<String>>> = match options.sort {
-        empty if empty.is_empty() => None,
+        empty if empty.is_empty() => Some(
+            DEFAULT_SORT
+                .iter()
+                .map(|&slice| Some(slice.to_string()))
+                .collect(),
```

Comment:
it's same thing in 10 places. replace the const with a `OnceCell` so it has the same value as this evaluated. you might need https://crates.io/crates/once_cell for it.